docs/API.md

```diff
@@ -1584,42 +1582,42 @@ If the `batch_id` is undefined the following error is returned:
 ### API method: `version_info`
 
 **Deprecated** (planned removal: v2024.1).
-Same as [`system_versions`][system_versions].
+Same as [`system_versions`][API system_versions].
 
 
 ### API method: `profile_names`
 
 **Deprecated** (planned removal: v2024.1).
-Same as [`conf_profiles`][conf_profiles].
+Same as [`conf_profiles`][API conf_profiles].
 
 
 ### API method: `get_language_tags`
 
 **Deprecated** (planned removal: v2024.1).
-Same as [`conf_languages`][conf_languages].
+Same as [`conf_languages`][API conf_languages].
```

Comment:
The new `get_language_tags` does not give the same result as the old `conf_languages`. Should it really say "same as" since it is not totally the same? Maybe "replaced by" or "use instead"?

Reply:
Updated using "Replaced by".